opacus/utils/uniform_sampler.py

```diff
@@ -0,0 +1,34 @@
+import torch
+from torch.utils.data import Sampler
+
+
+class UniformWithReplacementSampler(Sampler):
```

Comment:
This component is almost good to go! Let's add a docstring to this so that it's well documented.